api/app/signals/apps/feedback/managers.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.gis.db import models
+
+
+class FeedbackManager(models.Manager):
+    def request_feedback(self, signal):
+        from .models import Feedback
```

Comment:
You can also create a Feeback instance by calling `self.create(_signal=signal)`.

Reply:
will fix